dockerfiles/Dockerfile.android-builder

```diff
@@ -1,43 +1,8 @@
-FROM openjdk:8-jdk-stretch
-
-RUN apt-get update
-RUN apt-get install -y gradle jq
-
-WORKDIR /sdk
-RUN wget https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip -q
-RUN unzip -q sdk-tools-linux-4333796.zip
-
-ENV PATH="${PATH}:/sdk/tools:/sdk/tools/bin"
-
-RUN yes | sdkmanager "platform-tools"
-
-ENV PATH="${PATH}:/sdk/platform-tools"
-ENV ANDROID_HOME="/sdk/"
+ARG BRANCH_NAME
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/android:ci-${BRANCH_NAME} as android
 
 WORKDIR /app
 
-COPY gradlew build.gradle settings.gradle gradle.properties /app/
-COPY gradle/ /app/gradle/
-COPY bugsnag-android/ bugsnag-android/
-COPY bugsnag-android-ndk/ bugsnag-android-ndk/
-COPY bugsnag-plugin-android-anr/ bugsnag-plugin-android-anr/
-COPY bugsnag-android-core/ bugsnag-android-core/
-COPY bugsnag-plugin-android-ndk/ bugsnag-plugin-android-ndk/
-COPY bugsnag-plugin-react-native/ bugsnag-plugin-react-native/
-COPY examples/ examples/
-COPY mazerunner-scenarios/ mazerunner-scenarios/
-COPY scripts/ scripts/
-
-RUN scripts/install-ndk.sh
-
-ENV ANDROID_NDK_HOME=${ANDROID_HOME}/ndk-bundle
-
-ENV GRADLE_OPTS="-Dorg.gradle.daemon=false"
-
-RUN ./gradlew
-
-# Everything above this point should be derived from android-base
-
 RUN apt-get install maven
 
 # Setup signing
```

Comment:
The signing appears to be duplicated between the sizer + builder images. Not a show-stopper, but if it's possible to extract the two out to some common script it would make things easier to maintain.

Reply:
Have now pushed the signing down one level into the end of the CI image build.